wiring/inc/spark_wiring_cellular_printable.h

```diff
@@ -58,6 +51,13 @@ class CellularSignal : public particle::Signal, public Printable {
 
     virtual size_t printTo(Print& p) const;
 
+    virtual bool isAccessTechnologyValid() const;
+    virtual bool isSignalStrengthValid() const;
+    virtual bool isSignalQualityValid() const;
+    virtual bool isSignalQualitySupported() const;
```

Comment:
This class is called `CellularSignal` so there's no need to have the word `Signal` in its method names. The existing getter methods for the strength and quality values don't have `Signal` in their names as well.